nailgun/entities.py

```diff
@@ -3461,6 +3461,11 @@ def __init__(self, server_config=None, **kwargs):
             'docker_upstream_name': entity_fields.StringField(
                 default='busybox'
             ),
+            'download_policy': entity_fields.StringField(
+                choices=('background', 'immediate', 'on_demand'),
+                default='immediate',
+                required=True,
```

Comment:
This should be applicable for yum only repos.  May be we should set `required` flag only if `content_type == `yum``.